src/main/java/com/eerussianguy/firmalife/registry/ModRegistry.java

```diff
@@ -103,11 +123,23 @@ public static void registerItems(RegistryEvent.Register<Item> event) {
         easyItems.add(register(r, "milk_chocolate_blend", new ItemMisc(Size.SMALL, Weight.LIGHT), CT_MISC));
         easyItems.add(register(r, "white_chocolate_blend", new ItemMisc(Size.SMALL, Weight.LIGHT), CT_MISC));
         easyItems.add(register(r, "peel", new ItemMisc(Size.LARGE, Weight.VERY_HEAVY), CT_MISC));
-        allEasyItems = easyItems.build();
 
         ModRegistry.getAllIBs().forEach((x) -> {
             registerIB(r, x);
         });
+
+        for(CropFL crop : CropFL.values())
```

Comment:
I'm not sure why this is all being done here instead of with our register method that the other items use (it's named generically in a lot of cases but you should be able to use it fine) if that method doesn't have the bits you need I would rather you either make a new one or modify the existing one. It's just a lot of clutter for a class that's already bogged down